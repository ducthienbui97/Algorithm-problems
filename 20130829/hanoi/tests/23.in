3
23
AB AC CA BC CB BA
23
CB BC BA CA AC AB
23
BA CA AB BC CB AC
