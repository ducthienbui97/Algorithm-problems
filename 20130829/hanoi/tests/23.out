8388607
31381059609
62762119217
